python/k8s.py

```diff
@@ -303,6 +303,22 @@ def install_cert_manager(version):
         f.close()
         config.kubectl.call(['apply', '-n', 'cert-manager', '-f', f.name])
 
+@k8s.command(flowdepends=['k8s.create-cluster'])
+def install_dnsmasq():
+    """Install a dnsmasq server resolving *.localhost to 127.0.0.1. Supported OS: macOS."""
+    if sys.platform == "darwin":
+        call(['brew', 'install', 'dnsmasq'])
+        brew_prefix = check_output(['brew', '--prefix']).rstrip("\n")
+        with open(brew_prefix + "/etc/dnsmasq.conf", "r+") as f:
+            line_found = any("address=/localhost/127.0.0.1" in line for line in f)
+            if not line_found:
+                f.seek(0, os.SEEK_END)
+                f.write("\naddress=/localhost/127.0.0.1\n")
+        call(['sudo', 'brew', 'services', 'restart', 'dnsmasq'])
```

Comment:
brew run as root?

Reply:
You shouldn't use brew as root to install package but you have to run as root to restart services. 

Extracted from install log:

> [...]
> To restart dnsmasq after an upgrade:
>   sudo brew services restart dnsmasq
> Or, if you don't want/need a background service you can just run:
>   /usr/local/opt/dnsmasq/sbin/dnsmasq --keep-in-foreground -C /usr/local/etc/dnsmasq.conf -7 /usr/local/etc/dnsmasq.d,*.conf
> [...]